src/modules/75-cf/pages/feature-flags/components/__tests__/FlagTableFilters.test.tsx

```diff
@@ -40,7 +44,7 @@ const permanentFlagsFilter = {
 
 describe('FlagTableFilters', () => {
   beforeEach(() => {
-    flagEnabled = true
+    ;(flagEnabled = true), (archivingFFEnabled = false)
```

Comment:
is the semicolon at the start here right?

Reply:
yeaaah prettier added this automatically

Reply:
That's the downside of using JS without semicolons... sometimes they're needed and can't reliably be auto-injected at runtime. Though in this case a line break between seems fine in my IDE.
```suggestion
    flagEnabled = true
    archivingFFEnabled = false
```